### ISSUE TITLE
World space / nearest neighbour UV transfer?

### ISSUE THREAD

**Issue by polarathene** [nutti/Magic-UV#67]
I have a large 1.5 mil triangle mesh. I found it easier to unwrap first then slice up for game engine into smaller pieces for optimizations. Unfortuantely, the slicing process(Mesh mixer plane cut) due to introducing new triangles by creating straight line cuts, has destroyed the UV(or it doesn't care for keeping them since it's mostly for 3D printing). The topology is no longer identifical, but I would say 95% of the verts/edges overlap, so while vertex order is not the same anymore, it should be possible to some how transfer UV from one model to the other?

Going over the wiki, since topology is not identical it doesn't seem like Magic-UV permits this kind of transfer?

**Comment by schroef** [comment]
There is a modifier i believe what can do that. SOmething with data transfer. It project the UV from one mesh to another. PErhaps you can also try UV>transfer

**Comment by nutti** [comment]
@polarathene 

If the topology is not same among meshes, it may not work correctly in Magic UV.
however, you can copy/paste UV if you repeat trying to copy/paste each face.

**Comment by polarathene** [comment]
The mesh is 1.5 Million triangles, I don't think I can approach it with single copy/paste of each face :P It's just under 1k UV islands unwrapped.

I managed to solve it with Maya's very flexible `Mesh -> Transfer Attributes(options)` feature which can transfer properties like UVs via various spaces including local, world and topological. I used World Space Transferring from one single mesh to a sliced up mesh of 140 pieces.
